drivers_linux.go

```diff
@@ -17,3 +20,15 @@ func getInitializers() []initializer {
 		{overlay.Init, "overlay"},
 	}
 }
+
+func initIpams(ic ipamapi.Callback, ds interface{}) error {
```

Comment:
Can you please move this drivers.go? IPAMs are OS agnostic but if you want to do it in OS specific way you may need to follow the same design as how network drivers are initialized. Currently this will fail to build for windows and freebsd.